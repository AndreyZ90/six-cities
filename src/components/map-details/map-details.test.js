import React from 'react';
import TestRenderer from 'react-test-renderer';
import leaflet from 'leaflet';

import MapDetails from '@/components/map-details/map-details';

leaflet.map = () => {};

const geoCoords = [[23.1, 22.6], [12.9, 30.1]];
const center = [89.21, 78.12];
const zoom = 10;

describe(`MapDetails component snapshot`, () => {
  test(`Should correctly render MapDetails component`, () => {
    const tree = TestRenderer.create(
        <MapDetails
          geoCoords={geoCoords}
          center={center}
          zoom={zoom}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
