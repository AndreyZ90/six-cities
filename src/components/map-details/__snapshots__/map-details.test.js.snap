// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapDetails component snapshot Should correctly render MapDetails component 1`] = `
<section
  className="property__map map"
/>
`;
